helper_funcs.py

```diff
@@ -33,11 +42,28 @@
     "delayspectrum": ("delayspectrum_", ".h5"),
     "delayspectrum_hpf": ("delayspectrum_hpf_", ".h5"),
     "sensitivity": ("sensitivity_", ".h5"),
-    "rfi_mask": ("rfi_mask_", ".h5"),
+    "chisq": ("chisq_", ".h5"),
+    "power": ("lowpass_power_2cyl_", ".h5"),
+    "chisq_mask": {"rfi_mask_chisq_", ".h5"},
+    "stokesi_mask": ("rfi_mask_stokesi_", ".h5"),
+    "sens_mask": ("rfi_mask_sensitivity_", ".h5"),
+    "fact_mask": ("rfi_mask_factorized_", ".h5"),
     "sourceflux": ("sourceflux_", "_bright.h5"),
 }
 
 
+def _fail_quietly(func):
+    """Just log any exceptions."""
+
+    def wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except Exception as err:
+            logger.debug(f"Function {func.__name__} failed with error:\n{err}")
```

Comment:
I'd recommend an explicit `return None` here to make it clear that this is what the wrapped function will effectively return in this case.